front/app/containers/ProjectsShowPage/shared/header/ProjectHeader.tsx

```diff
@@ -90,17 +101,26 @@ const ProjectHeader = memo<Props & InjectedIntlProps>(
       return (
         <Container className={className || ''}>
           <ContentContainer maxWidth={maxPageWidth}>
-            {userCanEditProject && (
-              <EditButton
-                icon="edit"
-                locale={locale}
-                linkTo={`/admin/projects/${project.id}/edit`}
-                buttonStyle="secondary"
-                padding="5px 8px"
-              >
-                {formatMessage(messages.editProject)}
-              </EditButton>
-            )}
+            <TopBar>
```

Comment:
Might make sense to add a conditional check around TopBar (`{(projectFolderId || userCanEditProject) && ... }`) if you decide to give it some bottom margin, so that margin doesn't get applied to an empty element

Reply:
Just to be clear, before the `GoBackButton` was implemented there was no bottom margin when the edit button was shown (`if (userCanEditProject)`). Should I only add the margin if `GoBackButton` is visible? Or when either `GoBackButton` or the edit button (or both) are visible?

Reply:
I’d say the later, as the edit button by itself could also use the margin to boost its visibility.